src/brevitas/graph/equalize.py

```diff
@@ -694,82 +820,30 @@ def _no_equalize():
         module = region.get_module_from_name(name)
         maybe_allocate_params_accelerate(module)
 
-    act_sink_axes = {}
-    act_sources_axes = {}
     single_module = region.get_module_from_name(next(iter(region.sinks_names)))
     device = next(single_module.parameters()).device
     dtype = next(single_module.parameters()).dtype
 
-    src_axes = {}
-    for name, indexes in region.srcs.items():
-        module = region.get_module_from_name(name)
-        # If module is not supported, do not perform graph equalization
-        weight_axis = _get_output_axis(module)
-        act_sources_axes[name] = _get_act_axis(module)
-
-        if isinstance(module, nn.MultiheadAttention):
-            module = module.out_proj
-
-        src_axes[name] = EqualizationSourceWrapper(
-            module=module,
-            weight_tensor_name="weight",
-            weight_axis=weight_axis,
-            equalization_indexes=indexes,
-            bias_tensor_name="bias",
-            bias_axis=0,
-        )
-
-    sink_axes = {}
-    for name, indexes in region.sinks.items():
-        module = region.get_module_from_name(name)
-        weight_axis = _get_input_axis(module)
-        act_sink_axes[name] = _get_act_axis(module)
-        # For MultiheadAttention, we support only self-attention
-        # For sinks, we only need to modify the weight but not the bias
-        if isinstance(module, nn.MultiheadAttention) and module.in_proj_weight is not None:
-            # The weight attribute to equalize in nn.MultiheadAttention sinks is named "in_proj_weight"
-            weight_tensor_name = "in_proj_weight"
-        elif isinstance(module, nn.MultiheadAttention) and module.in_proj_weight is None:
-            return _no_equalize()
-        else:
-            weight_tensor_name = "weight"
-
-        sink_axes[name] = EqualizationSinkWrapper(
-            module=module,
-            weight_tensor_name=weight_tensor_name,
-            weight_axis=weight_axis,
-            equalization_indexes=indexes,
-        )
-
     # Check if any of the axis is None, which means that the module is not supported.
     # In that case, do not perform graph equalization
-    axes_to_check = [m.weight_axis for m in list(src_axes.values()) + list(sink_axes.values())]
+    axes_to_check = [
```

Comment:
No need to cast to list if using `itertools.chain`:
```suggestion
    axes_to_check =[m.weight_axis for m in chain(region.srcs.values(), region.sinks.values())]
```